sdk/cosmos/azure-cosmos-encryption/pom.xml

```diff
@@ -55,7 +55,7 @@ Licensed under the MIT License.
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-identity</artifactId>
-      <version>1.2.0-beta.1</version> <!-- {x-version-update;com.azure:azure-identity;current} -->
+      <version>1.2.0-beta.2</version> <!-- {x-version-update;com.azure:azure-identity;current} -->
```

Comment:
@moderakh @kushagraThapar 
Why does azure-cosmos-encryption-module take a direct dependency on Identity SDK ?
We should change it to test dependency and switch to `<!-- {x-version-update;com.azure:azure-identity;dependency} -->` tag.